app/controllers/api/v1/google_keys_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::GoogleKeysController < ApplicationController
+  def index
+    @google_key = Rails.application.credentials.google_api_token[:token]
+    render json: {
+      data: {
+        key: @google_key
+      }
+    }
+  end
+end
```

Comment:
what is the use of this API?

Reply:
For google map sa front end ma'am, para di po nakalabas sa code, nilagay ko nalang po sa credential ng back end

Reply:
> what is the use of this API?

For google map sa front end ma'am, para di po nakalabas sa code, nilagay ko nalang po sa credential ng back end

Reply:
Yah but you're actually exposing your token to the public. Maybe you can try other implementation

Reply:
> Yah but you're actually exposing your token to the public. Maybe you can try other implementation

okay po ma'am try po ako other way